File: program21.c
///////////////////////////////////////////////////////
//
//  File name :     program21.c
//  Descreption :   Used to Check Even or Odd
//  Author :        Vaibhav
//  Date :          11/05/2025
//
///////////////////////////////////////////////////////

#include<stdio.h>
#include<stdbool.h>

bool CheckEven(int iNo)
{
    return((iNo % 2) == 0);
}

int main()
{
    int iNo = 0;
    bool bRet = false;

    printf("_____________________________________________________________________\n");
    printf("--------- This program is used to Check Number is Even or Odd ---------\n");
    printf("_____________________________________________________________________\n");

    printf("Enter a number: ");
    scanf("%d",&iNo);

    bRet = CheckEven(iNo);

    if(bRet == true)
    {
        printf("%d is Even Number...!",iNo);
    }
    else
    {
        printf("%d is Odd Number...!",iNo);
    }
    printf("\n_____________________________________________________________________");
    return 0;
}

File: Java/program21.java
////////////////////////////////////////////////////
//
//  File name   :   program21.java
//  Description :   Check Even Number with using bool return value function
//  Author      :   Vaibhav Patil
//  Date        :   18/05/2025

import java.util.Scanner;

class CheckEven
{
    public boolean Check(int iNo)
    {
        return ((iNo % 2) == 0);
    }
}
class program21
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0;
        boolean bRet = false;

        System.out.println("Enter the number: ");
        iValue = sobj.nextInt();

        CheckEven dobj = new CheckEven();
        bRet = dobj.Check(iValue);

        if(bRet == true)
        {
            System.out.println(iValue+" is Even Number");
        }
        else
        {
            System.out.println(iValue+" is Odd Number");
        }
    }
}
